src/vs/code/electron-sandbox/issue/issueReporterMain.ts

```diff
@@ -740,7 +740,7 @@ export class IssueReporter extends Disposable {
 				show(extensionsBlock);
 			}
 			reset(descriptionTitle, localize('stepsToReproduce', "Steps to Reproduce"), $('span.required-input', undefined, '*'));
-			reset(descriptionSubtitle, localize('bugDescription', "Share the steps needed to reliably reproduce the problem. Please include actual and expected results. We support GitHub-flavored Markdown. You will be able to edit your issue and add screenshots when we preview it on GitHub."));
+			reset(descriptionSubtitle, localize('bugDescription', "Share the steps needed to reliably reproduce the problem.Please include actual and expected results.We support GitHub - flavored Markdown.You will be able to edit your issue and add screenshots when we preview it on GitHub. {0}", this.issueReporterModel.getData().versionInfo.vscodeVersion.includes('insiders') ? '' : 'Please ensure that this issue hasn\'t already been resolved on an insiders build before submitting.'));
```

Comment:
This will break the current translation. Also check that spaces are like in the original text.